### PR TITLE
Revise SVG Icons to use Icons import or css where necessary  

### PR DESCRIPTION
**Pull request by stevedya**
Addresses #7511: Replaced instances of $font-wagtail-icons being used for icons and left instances that would be overwritten by the 2022 page editor UI updates.

All screenshots are from testing in wagtail using Mac OS Big Sur v11.6.2 on these browsers: 
- Chrome v98.0.4
- Safari v15.2

### Icons Replaced For:
** Help Blocks ** 
Screen Shot 2022-02-28 at 3 47 27 PM

**Header Search Field Icon found in: snippets / redirects / shared header / model admin pages**
Screen Shot 2022-02-24 at 4 06 46 PM

**Error message X icon on fields** 
Screen Shot 2022-02-22 at 10 53 50 AM

**Checkboxes / Radio buttons**
Screen Shot 2022-03-02 at 12 45 52 PM
Screen Shot 2022-02-24 at 4 26 57 PM

**Tags** 
Screen Shot 2022-02-23 at 11 22 31 PM

**Select locale in Page explorer header** 
Locale Select Icon

**Button long running spinner** 
Screen Shot 2022-02-28 at 3 56 42 PM

**Comment add Icon** 
Replaced Icon in React with the reusable `<Icon>` component
Screen Shot 2022-03-02 at 1 04 39 PM

**Globe icon on explore page title wrappers**
Screen Shot 2022-03-03 at 2 04 29 PM

**chevron arrows on sortable table header labels**
Screen Shot 2022-03-03 at 1 45 53 PM

### Icons marked to be reformatted:
- Page editor Field blocks such as collapse, arrow down on Multifield panels, Date and View Icons on panels in editor settings panels
- Form fields with icons before text input (date, time, link)
- Date picker icons

### CODE REVIEW

**Comment by thibaudcolas** [comment]
@stevedya thank you for providing screenshost! Can you fix the failing CI tests, and describe in your PR description how you’ve tested this so far (which browsers in particular)?

File: wagtail/admin/static_src/wagtailadmin/scss/layouts/page-editor.scss [hunk 1]
```diff
@@ -413,15 +422,21 @@
...
     }
 
     &.collapsed {
       .title-wrapper {
         &:before {
-          content: map.get($icons, 'collapse-down');
+          content: map.get(
+            $icons,
+            'collapse-down'
+          ); // UI Redesign: To be removed in page editor redesign
```
[the remark this answers is not in the document]
Comment (stevedya) [reply wagtail/admin/static_src/wagtailadmin/scss/layouts/page-editor.scss:439]: Good suggestion!

**Comment by stevedya** [comment]
@thibaudcolas I have resolved that test now and just put a quick line about the browsers I've used to test. Let me know if there should be other tests taken for a PR of this magnitude 😄 . I'll re request review for now

**Review by thibaudcolas — CHANGES_REQUESTED** [review]
Looking good! With the exception of the listing changes, which unfortunately are still only partly there. I see you’ve re-added them in SVG, but they don’t update as expected based on the filtering status (we’d need to update the corresponding JS), and we’d need to update the tests as well.

Since this is all in separate files from the rest, I’ll revert those files back and merge the rest.

**Review by thibaudcolas — APPROVED** [review]
Tested with Firefox 97, Chrome 101, Safari 15.2. I’ve added separate line items for checkbox fixes only, since those are the only ones that will be different for end users. We’ll make a note about the icon implementation change once it’s completed throughout the admin.

**Pull request merged**